Dockerfile

```diff
@@ -35,6 +35,8 @@ ADD config/examples/*.yml config/
 # Needed for Sphinx ODBC
 ADD config/oracle/odbc*.ini /etc/
 
+ENTRYPOINT ["xvfb-run", "--server-args", "-screen 0 1280x1024x24"]
```

Comment:
why do we need this? 😄

Reply:
Oooops, error dealing with conflicts, sorry 🙈 , I will remove it 💪